org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/extensions/contentmodel/ContentModelPlugin.java

```diff
@@ -195,6 +196,8 @@ public void start(InitializeParams params, XMLExtensionsRegistry registry) {
 					new XMLValidationAllFilesCommand(contentModelManager, documentProvider, validationService));
 			commandService.registerCommand(AssociateGrammarCommand.COMMAND_ID,
 					new AssociateGrammarCommand(documentProvider));
+			commandService.registerCommand(CheckBoundGrammarCommand.COMMAND_ID,
+					new CheckBoundGrammarCommand(documentProvider));

```

Comment:
Even if it's not really used for the moment, please unregister the command on the stop method.